### ISSUE TITLE
Low Performance from Bad FPS Limiter

### ISSUE THREAD

**Issue by SGNull** [SGNull88/Python-Particles#1]
As of right now, the simulation can run with about 20-30 particles before some serious lag sets in. This is not a huge issue at the moment, as this is sufficient for testing purposes, but it will be eventually. We're shooting for 500+ particles here.

I suspect the problem lies with the graphics.py package, so I will need to look for a better package to use for this project.

**Comment by SGNull** [comment]
This issue was actually discovered about 7 hours ago. Not sure why it wasn't added to the repo.

**Comment by SGNull** [comment]
It appears that the issue is not with the graphics.py package, but with the way that the program handles FPS.

Currently the program pauses for a certain period of time, 1/SIM_FPS to be exact, between drawing the graphics. However, this pause is also placed between simulating the particles. This means that the whole program is stalled for an extra 1/SIM_FPS on top of however long it already takes to simulate.

I figured this out after thinking about the issue for a bit. I then set the SIM_FPS to 3000 and placed 200 particles in the Universe. And, unsurprisingly, it ran smoothly.
